packages/grafana-ui/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -46,14 +46,19 @@ export class ErrorBoundary extends PureComponent<Props, State> {
   }
 }
 
-interface WithAlertBoxProps {
+export interface ErrorBoundaryProps {
```

Comment:
I think we might need to add a comment with at least the `@public` attribute above this interface as well.

Reply:
done, thanks :)